# Include the system typedef rather than redefining u_int, which breaks
# sys/types.h on Tiger.
# random.h: error: 'u_int' has not been declared

--- lib/util.cpp.orig	2021-09-14 20:31:27.000000000 +0800
+++ lib/util.cpp	2022-03-25 01:56:45.000000000 +0800
@@ -18,6 +18,7 @@
 	#endif
 	#if HAVE_GETENTROPY
 		#ifdef __APPLE__
+			#include <sys/types.h>
 			#include <Availability.h>
 			#include <sys/random.h>
 		#endif
